src/applications/personalization/dashboard-2/components/health-care/HealthCare.jsx

```diff
@@ -83,7 +103,32 @@ const mapStateToProps = state => {
   const canAccessRx = profileState.services.includes('rx');
   const prescriptions = rxState.prescriptions?.items;
 
+  const cernerAppointmentFacilities = selectCernerAppointmentsFacilities(state);
+  const cernerMessagingFacilities = selectCernerMessagingFacilities(state);
+  const cernerPrescriptionFacilities = selectCernerRxFacilities(state);
+
+  const appointmentFacilityNames =
+    cernerAppointmentFacilities?.map(facility =>
+      getMedicalCenterNameByID(facility.facilityId),
+    ) || [];
+  const messagingFacilityNames =
+    cernerMessagingFacilities?.map(facility =>
+      getMedicalCenterNameByID(facility.facilityId),
+    ) || [];
+  const prescriptionFacilityNames =
+    cernerPrescriptionFacilities?.map(facility =>
+      getMedicalCenterNameByID(facility.facilityId),
+    ) || [];
+
+  const facilityNames = uniq([
```

Comment:
you could also do this without lodash :)

`new Set([...array1, ...array2, ...array3])`

And it looks like you could use a Set in place of Array with the FacilityList where this prop will eventually be used.

Reply:
Updated! I did need to convert the Set back into an array.

```
const facilityNames = [
    ...new Set([
      ...appointmentFacilityNames,
      ...messagingFacilityNames,
      ...prescriptionFacilityNames,
    ]),
  ];
```